activefires_pp/spatiotemporal_alarm_filtering.py

```diff
@@ -180,6 +181,9 @@ def run(self):
                 elif msg.type not in ['file', 'collection', 'dataset']:
                     LOG.debug("Message type not supported: %s", str(msg.type))
                     continue
+                elif product_list and msg.data.get('product') not in product_list:
```

Comment:
what if there is not product in the message? is None a possible value in the product_list?

Reply:
Ah good point! `None` is theoretically a possible name for a product, and then if the message does not contain the product name any product would pass through and give rise to a notification.

Reply:
Now I have changed so no product name in the message will check for products named "unknown" against the configured product names. You think this is better and sufficient?